trunk/SUAVE/Analyses/Stability/Fidelity_Zero.py

```diff
@@ -1,7 +1,7 @@
 # Fidelity_Zero.py
```

Comment:
This is overwriting changes that mike made. We need to keep the one on the left.